### ISSUE TITLE
RCS1036 doesn't allow an empty line between switch cases that use braces

### ISSUE THREAD

**Issue by xt0rted** [JosefPihrt/Roslynator#201]
Sometimes I'll wrap the contents of my switch cases in braces if they're more than a couple lines. When I do I leave the blank line between the closing brace and the next case, but the `Remove redundant empty line` rule is triggering on these instances.

An example is
```csharp
switch (number)
{
    case 1:
        DoSomething();
        break;

    case 2:
    {
        DoThings();
        DoMoreThings();
        break;
    }

    case 3:
        DoOtherThings();
        break;
}
```

The line before `case 3:` is what the empty line rule says should be removed.
